imu_bno055/launch/imu.launch.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2018 Toyota Research Institute.  All rights reserved.
+#
+# IF WE RELEASE THIS CODE, WE MAY USE THE FOLLOWING BOILERPLATE:
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Hmmm, if we're trying to upstream this, then this boilerplate stuff will have to be  removed... probably not suggesting Apache if the original package is MIT

Reply:
Right, we'll want to stick with the original license.  Also, I noticed that upstream changed to the BSD license a few days ago, so we'll probably want to pull that in and stick with BSD as well.

Reply:
@cuixiongyi you should probably merge in the upstream changes before we move with our internal open source process - we'll need to make sure the lawyers see the actual license that the software is going to be released under.